plenigo_plugin/PlenigoShortcodeManager.php

```diff
@@ -20,705 +20,741 @@
+			       . ' class="button button-primary ' . $cssClass . '" '
+			       . ' value="' . $btnTitle . '" '
+			       . ' onclick="' . $btnOnClick . '" />';
+		} else { //Return the content untouched
+			return do_shortcode( $content );
+		}
+	}
+
+	public function plenigo_handle_content_shortcode( $atts, $content = null, $tag = null ) {
+		$a = shortcode_atts( array(
+			'prod_id' => "",
+			'class'   => "",
+		), $atts );
+
+		$cssClass     = $a['class'];
+		$prodId       = $a['prod_id'];
+		$showContent  = false;
+		$returnString = "";
+		if ( is_null( $content ) || $content === false || ! is_string( $content ) ) {
+			$content = "";
+		}
+
+		$isBought = ( $prodId !== "" && PlenigoSDKManager::get()->plenigo_bought( explode( ',', $prodId ) ) );
+
+		if ( $tag == 'pl_content_show' ) {
+			if ( $isBought ) {
+				$showContent = true;
+			} else {
+				$showContent = false;
+			}
+		}
+		if ( $tag == 'pl_content_hide' ) {
+			if ( $isBought ) {
+				$showContent = false;
+			} else {
+				$showContent = true;
+			}
+		}
+		if ( $showContent ) {
+			if ( $cssClass != "" ) {
+				$returnString .= '<div class="' . $cssClass . '">';
+			}
+
+			$returnString .= do_shortcode( $content );
+
+			if ( $cssClass != "" ) {
+				$returnString .= '</div>';
+			}
+		}
+
+		return $returnString;
+	}
+
+	/**
+	 * This shortcode allows to show the user profile template with the data provided
+	 * from the plenigo SDK for the current, logged in, user. The content of the shortcode
+	 * can be used to customize the message for logged out users or users that doesnt have
+	 * plenigo information attached to it.
+	 *
+	 * @param  array $atts an associative array of attributes, or an empty string if no attributes are given
+	 * @param  string $content the enclosed content (if the shortcode is used in its enclosing form)
+	 * @param  string $tag the shortcode tag, useful for shared callback functions
+	 *
+	 * @return string the contents of the shortcode or the user profile
+	 */
+	public function plenigo_handle_user_shortcode( $atts, $content = null, $tag = null ) {
+		$a = shortcode_atts( array(
+			'class' => "",
+		), $atts );
+
+		$loggedIn = UserService::isLoggedIn();
+		//If it's logged in the we should the user profile template
+		if ( $loggedIn ) {
+			$user         = PlenigoSDKManager::get()->getSessionValue( "plenigo_user_data" );
+			$userLoggedIn = UserService::getCustomerInfo();
+			if ( ! is_null( $user ) && ! is_null( $userLoggedIn ) ) {
+				return $this->get_profile_code( $user );
+			} else {
+				return do_shortcode( $content );
+			}
+		} else { // Else we show the shortcode contents to allow customize the logged out message
+			return do_shortcode( $content );
+		}
+	}
+
+	public function plenigo_handle_snippet_shortcode( $atts, $content = null, $tag = null ) {
+		if ( ! $this->isHomePage() ) {
+			$a           = shortcode_atts( array(
+				'name'         => "",
+				'redirectUrl'  => "",
+				redirectUrlBox => "no"
```

Comment:
Check formatting of the file, is 'redirectUrlBox' correct?

Reply:
fixed